src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudConfig.scala

```diff
@@ -80,6 +80,7 @@ object FireCloudConfig {
     lazy val whitelistBucket = sys.env.get("NIH_WHITELIST_BUCKET").get
     lazy val whitelistFile = nih.getString("whitelistFile")
     lazy val rawlsGroupName = nih.getString("rawlsGroupName")
+    lazy val dummyToken = sys.env.get("NIH_DUMMY_TOKEN")
```

Comment:
I assume that a corresponding change in https://github.com/broadinstitute/firecloud-develop/blob/local/configs/firecloud-orchestration/docker-compose.yaml will be necessary to support this?